.github/workflows/bazel.yml

```diff
@@ -142,9 +142,9 @@ jobs:
     needs: path_filter
     # Only run workflow if this is a scheduled run on master branch,
     # or a pull_request that skip-duplicate-action wants to run again.
-    if: |
-      (github.event_name == 'schedule' && github.ref == 'refs/heads/master') ||
-      needs.path_filter.outputs.files_changed == 'true'
+#    if: |
+#      (github.event_name == 'schedule' && github.ref == 'refs/heads/master') ||
+#      needs.path_filter.outputs.files_changed == 'true'
```

Comment:
I think these commented-out filters need to be reverted?

Reply:
Done, we commented these out temporarily to run the workflow on my fork.